src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -182,9 +182,12 @@ protected virtual void ProcessBaseline(IAnalysisContext context, T driverOptions
 
         protected virtual void PostLogFile(string postUri, string outputFilePath, IFileSystem fileSystem)
         {
-            PostLogFile(postUri, outputFilePath, fileSystem, new HttpClient())
-                .GetAwaiter()
-                .GetResult();
+            using (var httpClient = new HttpClient())
+            {
+                PostLogFile(postUri, outputFilePath, fileSystem, httpClient)
+                    .GetAwaiter()
+                    .GetResult();
+            }
```

Comment:
we need to use the old-style using to prevent it from being disposed before the method finishes.

Reply:
Why are we creating a new http client? It is because we only expect a single post, for a single log file?

Also, you didn't explain why we can't use the new style using syntax. Why is that?

using var httpClient = new HttpClient();
PostLogFile(postUri, outputFilePath, fileSystem, httpClient)
                    .GetAwaiter()
                    .GetResult();